<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow7</class>
 <widget class="QDialog" name="mainwindow7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_client">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>1153</width>
     <height>605</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ajouter client</string>
    </attribute>
    <widget class="QGroupBox" name="id">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>1011</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>ajoue</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>60</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>110</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>160</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>93</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>ajoute</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="Date">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2002</year>
        <month>2</month>
        <day>5</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Sexe">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Capteur Temp√©rature:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>supprimer client</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id_supp">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>121</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>240</y>
       <width>381</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>modifier client</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>320</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nouvelle Id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nouveau Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nouveau Prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_id">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_nom">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_prenom">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="old_id">
     <property name="geometry">
      <rect>
       <x>472</x>
       <y>320</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>afficher client</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_recherche">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_in">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>70</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_id">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>180</y>
       <width>116</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par id</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>220</y>
       <width>116</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par Date</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>377</y>
       <width>431</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Maps</string>
    </attribute>
    <widget class="QAxWidget" name="WebBrowser">
     <property name="control" stdset="0">
      <string>{8856f961-340a-11d0-a96b-00c04fd705a2}</string>
     </property>
     <property name="geometry" stdset="0">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>1141</width>
       <height>461</height>
      </rect>
     </property>
     <property name="enabled" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
